File: 2017/fall/8/week2/mario3.py
# Prints a positive number of question marks, as specified by user

from cs50 import get_int

# Prompt user for a positive number
while True:
    n = get_int("Positive number: ")
    if n > 0:
        break

# Print out that many bricks
for i in range(n):
    print("#")

File: 2017/fall/2/src2/mario3.c
// Prints a positive number of question marks, as specified by user

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt user for a positive number
    int n;
    do
    {
        n = get_int("Positive number: ");
    }
    while (n <= 0);

    // Print out that many bricks
    for (int i = 0; i < n; i++)
    {
        printf("#\n");
    }
}
